modules/appeals_api/app/controllers/appeals_api/schemas/shared_schemas_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class AppealsApi::Schemas::SharedSchemasController < AppealsApi::ApplicationController
+  skip_before_action :authenticate
+  before_action :check_schema_type
+
+  ACCEPTED_SCHEMA_TYPES = %w[
+    non_blank_string
+    address
+    date
+    phone
+    timezone
+  ].freeze
+
+  def show
+    render json: AppealsApi::Schemas::SchemaGetter.new(schema_type).call
```

Comment:
`SchemaGetter` doesn't feel quite right to me. `SchemaRetriever` or even just `Shared` or `SharedFile` (since the `Schemas` namespace is already there, indicating what we're working with) feels better.

Reply:
Ok. I love it lol. Retriever works too. I feel like the other titles don't actually represent what the service does. Any other thoughts?

Reply:
Many, but they're mostly just shuffling things around in different ways and not really able to come to a consensus so I won't bother trying 😆

Reply:
😆 Ok we can solve this via a call.